createReleaseAction/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Create Release'
```

Comment:
It is a bit confusing. Why not rename this file as `createReleaseAction.yml` and `action.yml` file as `changelogAction.yml`? And place them outside (i.e. not in a folder?)

Reply:
Because github actions will looks for `action.yml` file.